index.html

```diff
@@ -13,6 +13,7 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
+    <script src="index.js"></script>
```

Comment:
JS script needs to be at the end of the body tag not before. Adding it at the top isn't a good practice since it slows down HTML rendering time when site is first loaded